.github/workflows/tests.yml

```diff
@@ -0,0 +1,28 @@
+name: Unit Tests for Python 3.7, 3.8, 3.9
```

Comment:
Unit Tests run in Python 3.7, 3.8 and 3.9.
flake8 (and later mypy) will only run in Python 3.7 (see tox.ini)